job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/step/MultiJobStepContext.groovy

```diff
@@ -35,20 +39,29 @@ class MultiJobStepContext extends StepContext {
      * {@code continuationCondition} must be one of {@code 'SUCCESSFUL'}, {@code 'UNSTABLE'}, {@code 'COMPLETED'} or
      * {@code 'FAILURE'}. When version 1.16 or later of the MultiJob plugin is installed, {@code continuationCondition}
      * can also be set to {@code 'ALWAYS'}.
+     * {@code executionType} must be one of {@code 'PARALLEL'} or {@code 'SEQUENTIALLY'}
      */
-    void phase(String name, String continuationCondition, @DslContext(PhaseContext) Closure phaseClosure) {
-        PhaseContext phaseContext = new PhaseContext(jobManagement, item, name, continuationCondition)
+    void phase(
```

Comment:
This will break compatibility as it changes a signature. Leave the signature as it is, just adding a new method to `PhaseContext` is sufficient.